hack/install-osp.sh

```diff
@@ -113,9 +138,35 @@ EOF
 rollout_status "openshift-pipelines" "tekton-pipelines-controller"
 rollout_status "openshift-pipelines" "tekton-pipelines-webhook"
 
+install_serverless
+
+# wait until serverless operator is created
+echo "Waiting for OpenShift Serverless Operator to be created..."
```

Comment:
We may need to update the timeout to 3 or 4 minutes maybe.

Reply:
yep updated with 4m